docs/install.rst

```diff
@@ -49,7 +49,7 @@ or install it `editable <https://pip.pypa.io/en/stable/reference/pip_install/#ed
 so that you can do changes and submit patches.
 
 
-[Linux/macOS] Install via your package manager
+Install via your package manager - Linux/macOS
```

Comment:
Please revert this change, we don't want to break the existing hyperlink.

Reply:
If issue only is hyperlink breakage, I'll try to fix all hyperlink breakage.

Reply:
No, please revert this line as a whole.